src/utils/housesFilterArr.ts

```diff
@@ -0,0 +1 @@
+export const housesFilterArr = ['hasTitles', 'hasSeats', 'hasDiedOut'];
```

Comment:
```suggestion
export const housesFilters = ['hasTitles', 'hasSeats', 'hasDiedOut'];
```
Стараемся избегать упоминаний по типу Arr. То, что это массив должно подчеркиваться неймингом и типизацией.